tests/integration/sqs_work_dispatcher_tests.py

```diff
@@ -0,0 +1,522 @@
+import inspect
+import unittest
+import boto3
+import logging
+import multiprocessing as mp
+import os
+import signal
+import psutil as ps
+
+from random import randint
+from botocore.config import Config
+from botocore.exceptions import EndpointConnectionError, ClientError, NoCredentialsError, NoRegionError
+from dataactcore.aws.sqsHandler import sqs_queue
+from dataactvalidator.sqs_work_dispatcher import SQSWorkDispatcher, QueueWorkerProcessError
+from tests.integration.baseTestValidator import BaseTestValidator
+from time import sleep
+
+
+class SQSWorkDispatcherTests(BaseTestValidator):
+    def tearDown(self):
+        sqs_queue().purge()  # clear any lingering messages in the queue between tests
+        super().tearDown()
+
+    def test_default_dispatch_with_numeric_message_body_succeeds(self):
+        """SQSWorkDispatcher can execute work on a numeric message body successfully
+
+        - Given a numeric message body
```

Comment:
Please follow the style guide (applies to all tests with multiline comments)

Reply:
Done